_assets/js/pagination.js

```diff
@@ -1,7 +1,6 @@
-import renderSearchResults from './utils/renderSearchResults';
 import { NUMBER_OF_RESULTS, SEARCH_ENDPOINT, ACCESS_KEY, AFFILIATE } from './utils/constants';
 import renderSearchPage from './utils/renderSearchPage';
-import noResults from './templates/search/noResultsTemplate';
+import totalResults from "./templates/search/totalResultsTemplate";
```

Comment:
Does noResultsTemplate still get used somewhere?

Reply:
No, it should all be handled by the totalResultsTemplate.

Reply:
Cool!